e2e/ibc_to_cctp_test.go

```diff
@@ -169,3 +349,73 @@ func TestIbc(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, math.NewInt(10_000).String(), feeAmt.String())
 }
+
+func testIbcPassingWithoutActions(
+	t *testing.T,
+	ctx context.Context,
+	cdc codec.Codec,
+	s *Suite,
+	dstUsdcDenom string,
+	toOrbiterChannelID string,
+) {
+	amountToSend := math.NewInt(OneE6)
+
+	mintRecipient := testutil.RandomBytes(32)
+	destCaller := testutil.RandomBytes(32)
+	forwarding, err := forwardingtypes.NewCCTPForwarding(
+		uint32(0),
+		mintRecipient,
+		destCaller,
+		[]byte(""),
+	)
+	require.NoError(t, err)
+
+	p := core.PayloadWrapper{Orbiter: &core.Payload{
+		PreActions: []*core.Action{},
+		Forwarding: forwarding,
+	}}
+	payloadBz, err := types.MarshalJSON(cdc, &p)
+	require.NoError(t, err)
+
+	height, err := s.Chain.Height(ctx)
+	require.NoError(t, err)
+
+	transfer := ibc.WalletAmount{
+		Address: OrbiterModuleAddr,
+		Denom:   dstUsdcDenom,
+		Amount:  amountToSend,
+	}
+	_, err = s.IBC.CounterpartyChain.SendIBCTransfer(
+		ctx,
+		toOrbiterChannelID,
+		s.IBC.CounterpartySender.KeyName(),
+		transfer,
+		ibc.TransferOptions{
+			Memo: string(payloadBz),
+		},
+	)
+	require.NoError(t, err)
+	s.FlushRelayer(t, ctx, toOrbiterChannelID)
+
+	ibcHeight := s.GetIbcTransferBlockExecution(t, ctx, height)
+	txsResult := GetTxsResult(t, ctx, s.Chain.Validators[0], strconv.Itoa(int(ibcHeight)))
+	require.Equal(t, txsResult.TotalCount, uint64(1), "expected only one tx")
+
+	found, _ := SearchEvents(txsResult.Txs[0].Events, []string{
+		"circle.cctp.v1.DepositForBurn",
+	})
+	require.True(t, found)
+
+	resp, err := s.Chain.GetBalance(
+		ctx,
+		core.DustCollectorAddress.String(),
+		Usdc,
+	)
+	require.NoError(t, err)
+	require.Equal(
+		t,
+		math.NewInt(OneE6),
+		resp,
+		"expected the dust collector to have received orbiter module initial balance",
+	)
```

Comment:
could be good to use different sent amounts here to make sure that it's the correct balance that is left and not something else; would be easier to identify if the `amountToSend` and the dust collector balance were not the same